### PR TITLE
Modify styleguide to fit Mirego standards and add linters files

### PR DESCRIPTION
**Pull request by charlesdemers**
Sorry for the huge diff, there was a lot of little things that were itching me :smile: 

I invite you to read the styleguide in the rendered view and give me your feedback. I think it pretty much sums up what we do as of today.

I also added a `.jshintrc` I use in my projects and the `.jscs.json` is a copy of the Airbnb one except it’s modified for our styleguide obviously.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +40,7 @@
     + `null`
     + `undefined`
 
-    ```javascript
+    ```js
```
Comment (jfcartkeep) [inline README.md]: Vraiiimmmment picky/question pourquoi avoir remplacé les `javascript` par `js` ? Selon [ceci](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L1450-L1457), `js` est un alias. Ça serait pas mieux de toujours aller avec le terme primaire plutôt qu'un alias?

File: README.md [hunk 2]
```diff
@@ -1457,160 +1344,9 @@
...
-## Chat With Us About JavaScript
-
-  - Find us on [gitter](https://gitter.im/airbnb/javascript).
-
-## Contributors
-
-  - [View Contributors](https://github.com/airbnb/javascript/graphs/contributors)
-
-
-## License
```
Comment (remiprev) [inline README.md]: Should we keep some kind of license?

File: README.md (hunk 1 above)
Comment (charlesdemers) [inline README.md]: It’s just shorter to write when adding new examples. I found that in other repositories I was writing `js` too so... went with the consistency :smile:

File: README.md (hunk 2 above)
Comment (charlesdemers) [inline README.md]: I don’t know, maybe I could add the standard Mirego license file in another pull request?

File: README.md [hunk 3]
```diff
@@ -639,91 +599,32 @@
...
-    // bad
-    var active = true;  // is current tab
+  - Use `// FIXME` to annotate problems.
 
-    // good
-    // is current tab
-    var active = true;
-
-    // bad
-    function getType() {
```
Comment (simonprev) [inline README.md]: Why did you remove this section?

File: README.md (hunk 2 above)
Comment (remiprev) [inline README.md]: We should keep the existing license and add `Copyright (c) 2015 Mirego` under the `Copyright (c) 2014 Airbnb` line. We forked their repo, we can’t throw *their* license away :smile:

File: README.md (hunk 3 above)
Comment (charlesdemers) [inline README.md]: Well because we don’t tend to write a lot of comments... I don’t know if it’s worth the entire section explaining how to write single-line comments...

File: README.md [hunk 4]
```diff
@@ -1457,160 +1344,9 @@
...
-THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND,
-EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
-MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
-IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
-CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
-TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
-SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+    + [Phare](https://github.com/mirego/phare)
+    + [JSHint](http://www.jshint.com/) - [Mirego Style .jshintrc](https://github.com/mirego/javascript-style-guide/blob/master/linters/.jshintrc)
+    + [JSHint](http://jscs.info/) - [Mirego Style .jscs.json](https://github.com/mirego/javascript-style-guide/blob/master/linters/.jscs.json)
```
Comment (jfcartkeep) [inline README.md]: Should be `JSCS` not `JSHint` again :wink:

File: README.md (hunk 2 above)
Comment (charlesdemers) [inline README.md]: Don’t ask me, I’m no lawyer! :smile: I’ll add it back.

**Comment by jfcartkeep** [comment]
:+1:

File: README.md (hunk 4 above)
Comment (charlesdemers) [inline README.md]: Good catch, fixed!

**Comment by simonprev** [comment]
:+1:

**Comment by remiprev** [comment]
:+1:
